test/requirements.py

```diff
@@ -1344,6 +1344,10 @@ def psycopg_compatibility(self):
             ]
         )
 
+    @property
+    def psycopg_only_compatibility(self):
```

Comment:
I think we could rename the `psycopg_compatibility` to call it `any_psycopg_compatibility`, so that this can take that name